internal/commands/permission.go

```diff
@@ -47,14 +47,6 @@ func consistencyFromCmd(cmd *cobra.Command) (c *v1.Consistency, err error) {
 		c = &v1.Consistency{Requirement: &v1.Consistency_AtLeastAsFresh{AtLeastAsFresh: &v1.ZedToken{Token: atLeast}}}
 	}
 
-	// Deprecated (hidden) flag.
-	if revision := cobrautil.MustGetStringExpanded(cmd, "revision"); revision != "" {
```

Comment:
This is now covered by the consistency option; we want to get rid of the old way of doing things.